lib/models/mixins/consumable.js

```diff
@@ -3,12 +3,18 @@ module.exports = provider => superclass => class extends superclass {
     return [
       ...super.IN_PAYLOAD,
       'consumed',
+      'consumedCount',
     ];
   }
 
   consume() {
     provider.emit('token.consumed', this);
```

Comment:
Does this needs to be moved accordingly?

Reply:
no, this is statsd stuff we don't care about.